admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -293,6 +304,75 @@ On the one hand the memory usage is reduced, on the other hand there is more net
 We recommend using the option with a value high enough to process a large number of notifications, but not so large to crash the process.
 Between 200 and 500 should be fine, and we'll likely process all the notifications in one go.
 
+.. _password-options-label:
+
+Password Options
+----------------
+
+There are three ways to supply a password:
+
+#. Interactively in response to a password prompt.
+#. Sent as a parameter to the command, e.g., ``occ wnd:listen host share username password``.
+#. Read from a file, using the ``--password-file`` switch to specify the file to read from. 
+#. Using 3rd party software to store and fetch the password. When using this option, the 3rd party app needs to show the password as plaintext on standard output.
+
+basic example:
```

Comment:
These examples feels duplicated with the ones provided below in the "3rd party sotfware examples" section. It might be better to link to them from here.

In addition, the contents must be merged with the ones in the "wnd:listen" section because the information is duplicated there.

Reply:
so why not just leave them out?

I am not sure how to link in the rst doc, since I never got a proper introduction to it.

Also I am not sure what you mean exactly, I have no knowledge in the wnd.

If you could write what you want to go in to the docs and I implement it - that would be great.